### PR TITLE
use English message format instead of the dot notation

### PR DESCRIPTION
**Pull request by cheahuychou**
resolves: https://github.com/Originate/exosphere-sdk/issues/258

### CODE REVIEW

File: tweets-service/service.yml [hunk 1]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
...
 messages:
   receives:
-    - tweets.create
-    - tweets.list
-    - tweets.get-details
-    - tweets.update
-    - tweets.delete
+    - create tweets
+    - list tweets
+    - get tweets details
```
Comment (hugoesthere) [inline tweets-service/service.yml]: `get tweet details`

File: tweets-service/service.yml [hunk 2]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
 
 messages:
   receives:
-    - tweets.create
-    - tweets.list
-    - tweets.get-details
-    - tweets.update
-    - tweets.delete
+    - create tweets
```
Comment (hugoesthere) [inline tweets-service/service.yml]: `create tweet`

File: tweets-service/service.yml [hunk 3]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
...
+    - delete tweets
   sends:
-    - tweets.created
-    - tweets.listed
-    - tweets.details
-    - tweets.not-found
-    - tweets.updated
-    - tweets.deleted
-    - tweets.not-created
+    - tweets created
```
Comment (hugoesthere) [inline tweets-service/service.yml]: `tweet created`

File: tweets-service/service.yml [hunk 4]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
...
+    - tweets listed
+    - tweets details
```
Comment (hugoesthere) [inline tweets-service/service.yml]: `tweet details`

File: tweets-service/service.yml [hunk 5]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
...
+    - tweets not found
```
Comment (hugoesthere) [inline tweets-service/service.yml]: Does tweets-service look for a single tweet or multiple? If it's a single then this should be `tweet not found`

File: tweets-service/service.yml [hunk 6]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
...
+    - update tweetsd
```
Comment (hugoesthere) [inline tweets-service/service.yml]: This is saying that a tweet has been updated, no? So it should be `tweet updated`

File: tweets-service/service.yml [hunk 7]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
...
+    - tweets deleted
```
Comment (hugoesthere) [inline tweets-service/service.yml]: `tweet deleted` if this is only deleting one tweet at a time.

File: tweets-service/service.yml [hunk 8]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
...
+    - tweets not created
```
Comment (hugoesthere) [inline tweets-service/service.yml]: If this is referring to a single tweet please make it singular.

File: users-service/service.yml [hunk 9]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
...
 messages:
   receives:
-    - users.create
-    - users.list
-    - user.get-details
-    - user.update
-    - user.delete
+    - create users
+    - list users
+    - get users details
```
Comment (hugoesthere) [inline users-service/service.yml]: `get user details`, I think users service only lists the details of one user at a time.

File: users-service/service.yml [hunk 10]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
...
+    - update users
+    - delete users
```
Comment (hugoesthere) [inline users-service/service.yml]: These might be singular as well?

File: users-service/service.yml [hunk 11]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
 
 messages:
   receives:
-    - users.create
-    - users.list
-    - user.get-details
-    - user.update
-    - user.delete
+    - create users
```
Comment (hugoesthere) [inline users-service/service.yml]: `create user`?

File: users-service/service.yml [hunk 12]
```diff
@@ -11,19 +11,19 @@ tests: node_modules/.bin/cucumber-js
...
-    - user.updated
-    - user.deleted
-    - users.not-created
+    - users created
+    - users listed
+    - user details
+    - user not found
+    - user updated
+    - user deleted
+    - users not created
```
Comment (hugoesthere) [inline users-service/service.yml]: Make these the proper pluralization as well.

File: web-server/service.yml [hunk 13]
```diff
@@ -10,24 +10,24 @@ startup:
...
-    - tweets.deleted
+    - user details
+    - user not found
+    - user updated
+    - user deleted
+    - users listed
+    - users created
+    - tweets listed
+    - tweets created
+    - tweets deleted
```
Comment (hugoesthere) [inline web-server/service.yml]: Please make these the proper pluralization.

**Comment by cheahuychou** [comment]
@hugobho All comments addressed. Sorry, I really should've paid attention to what the code does before replacing anything.

**Review by hugoesthere — APPROVED** [review]
🎃

**Pull request closed without merging**